src/main/java/com/epam/ta/reportportal/dao/OAuthRegistrationRestrictionRepositoryCustom.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.ta.reportportal.dao;
+
+import com.epam.ta.reportportal.entity.oauth.OAuthRegistrationRestriction;
+
+import java.util.List;
+
+/**
+ * @author Anton Machulski
+ */
+public interface OAuthRegistrationRestrictionRepositoryCustom {
+	List<OAuthRegistrationRestriction> findByRegistrationId(String registrationId);
```

Comment:
i suppose such queries could be done with existed jpa query builder
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.query-lookup-strategies